apps/financial-aid/web-veita/src/components/ModalTypes/AcceptModal.tsx

```diff
@@ -173,11 +201,28 @@ const AcceptModal = ({
         />
       </Box>
 
+      {hasDecemberAid && (
+        <Box marginBottom={3}>
+          <NumberInput
+            label="Desember uppbót"
+            placeholder="Sláðu inn upphæð útborgunar"
```

Comment:
Should this copy be either:

'Sláðu inn upphæð'
or
'Sláðu inn upphæð uppbótar'?